newlib/libc/sys/hermit/include/sys/fcntl.h

```diff
@@ -16,6 +16,7 @@ __BEGIN_DECLS
 #define F_SETLKW 7
 
 #define FD_CLOEXEC 1
+#define AT_FDCWD -100
```

Comment:
I think we should use parens here. 🤔 


```suggestion
#define AT_FDCWD (-100)
```

Reply:
I added some parentheses